server/src/metadata.rs

```diff
@@ -748,4 +750,25 @@ pub mod entities_metadata_query {
             _ => vec![],
         }
     }
+
+    fn map_serlo_subjects_to_thumbnail(id: Option<&i32>) -> String {
+        let thumbnail_folder: String =
+            "https://raw.githubusercontent.com/serlo/frontend/staging/public/_assets/img/meta/"
+                .into();
+        let thumbnail_image = match id {
+            // Biologie (Schule)
+            Some(23362) => "biologie.png",
```

Comment:
I would define the mapping from the subject enum to thumbnail and not taxonomy ID to thumbnail. Otherwise we have implicitly duplicate code (mapping from taxonomy ID to subject)

Reply:
We decided against using AMB standard subject IDs for mapping the thumbnails, because of cases like sustainability
```rust
// Nachhaltigkeit => Biologie, Ethik (Schule)
17744 | 48416 | 242851 => vec![
    RawSubjectMetadata {
        id: "1001".to_string(),
        in_scheme: SchemeId::SchoolSubject,
    }
    .into(),
    RawSubjectMetadata {
        id: "1008".to_string(),
        in_scheme: SchemeId::SchoolSubject,
    }
    .into(),
],
```
, where the first AMB standard ID is actually biology, so if we were mapping from AMB standard IDs to thumbnails, we would be using the "biologie.png" thumbnail from sustainability, instead of "nachhaltigkeit.png" thumbnail.

In general, the images in [the provided repository](https://github.com/serlo/frontend/tree/staging/public/_assets/img/meta) are made for Serlo subjects, not for AMB subjects.

Taking this into account, do you still think that we should be mapping from AMB standard IDs to thumbnails?